src/test/java/com/fontys/api/service/TournamentServiceTest.java

```diff
@@ -109,4 +132,105 @@ private void createTournamentTestInvalid(String name, String description, Intege
             assertEquals(errorMessage, e.getMessage());
         }
     }
+
+    @Test
+    public void getAllTournaments()
+    {
+        List<Tournament> expectedTournaments = new ArrayList<>();
+        expectedTournaments.add(tournament1);
+        expectedTournaments.add(tournament2);
+        expectedTournaments.add(tournament3);
+        expectedTournaments.add(tournament4);
+        expectedTournaments.add(tournament5);
+        expectedTournaments.add(tournament6);
+
+        when(tournamentRepository.findAll()).thenReturn(expectedTournaments);
+
+        List<Tournament> actualTournaments = null;
+        try
+        {
+            actualTournaments = tournamentService.tournaments(null);
+        }
+        catch (InvalidAttributeValueException e)
+        {
+            fail();
+        }
+
+        assertEquals(expectedTournaments, actualTournaments);
+        Mockito.verify(tournamentRepository, times(1)).findAll();
+    }
+
+    @Test
+    public void getAllTournamentsByOwnerId()
+    {
+        List<Tournament> expectedTournaments = new ArrayList<>();
+        expectedTournaments.add(tournament1);
+        expectedTournaments.add(tournament2);
+        expectedTournaments.add(tournament3);
+
+        when(userRepository.findById(Mockito.anyInt())).thenReturn(Optional.of(user1));
+        when(tournamentRepository.findByOwner(Mockito.any(User.class))).thenReturn(expectedTournaments);
+
+        List<Tournament> actualTournaments = null;
+
+        try
+        {
+            actualTournaments = tournamentService.tournaments(user1.getId());
+        }
+        catch (InvalidAttributeValueException e)
+        {
+            fail();
+        }
+
+        assertEquals(expectedTournaments, actualTournaments);
+        verify(tournamentRepository, times(1)).findByOwner(user1);
+        verify(userRepository, times(1)).findById(user1.getId());
+    }
+
+    @Test
+    public void getAllTournamentsByOwnerIdNoTournaments()
```

Comment:
Isn't this an unnecessary test, as we already test getting all tournamentsByOwnerId?

Reply:
I think it's a nice way to test this unexpected scenario which is still valid. I think it's completely different from the tournamentsByOwnerId